artichoke-backend/Cargo.toml

```diff
@@ -29,9 +29,7 @@ spinoso-symbol = { version = "0.1", path = "../spinoso-symbol" }
 spinoso-time = { version = "0.1", path = "../spinoso-time", optional = true }
 
 [dependencies.onig]
-version = "6.0.0"
-git = "https://github.com/artichoke/rust-onig"
-rev = "v6.0.0-artichoke.2"
+version = "6.1.0"
```

Comment:
Thank you! I've had the 6.1.0 release email sitting in my inbox but hadn't got around to making the change yet.